### PR TITLE
Switched CardWrapper to MD3

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt [hunk 1]
```diff
@@ -42,10 +44,13 @@ fun BpkCardWrapper(
     corner: BpkCardCorner = BpkCardCorner.Small,
     elevation: BpkCardElevation = BpkCardElevation.Default,
 ) {
-    Card(
-        modifier = modifier.fillMaxWidth().border(width = BpkBorderSize.Lg, color = backgroundColor, shape = cardShape(corner)),
-        backgroundColor = backgroundColor,
+    Surface(
```
Comment (bvitaliyg) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt:47]: This component is more suited for this case that all these custom modifiers + card (with different colours and elevations

Comment (marianeum) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt:47]: this also had the default elevation before (which was wrong), but based on figma the card wrapper should have the same card elevation levels set

Comment (marianeum) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt:47]: correct. but in this case we're changing how it looks like already, so we might as well fix the behaviour at the same time

**Review by marianeum — COMMENTED** [review]
can you also regenerate the docs screenshots please?

**Comment by bvitaliyg** [comment]
@marianeum done

**Review by marianeum — APPROVED** [review]

**Pull request merged**